projects/element-ng/side-panel/si-side-panel.component.html

```diff
@@ -1,3 +1,10 @@
+@if (showBackdrop()) {
+  <div
+    class="modal-backdrop"
+    [@backdrop]="showBackdrop() ? 'show' : 'hide'"
```

Comment:
Angular animations are now legacy. Please use alternatives to built them: https://angular.dev/guide/animations/migration#